gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/GremlinServer.java

```diff
@@ -71,21 +74,33 @@
     private final EventLoopGroup workerGroup;
     private final ExecutorService gremlinExecutorService;
     private final ServerGremlinExecutor<EventLoopGroup> serverGremlinExecutor;
+    private final boolean isLinuxOS;
 
     /**
      * Construct a Gremlin Server instance from {@link Settings}.
      */
     public GremlinServer(final Settings settings) {
         this.settings = settings;
+        this.isLinuxOS = settings.useEpollEventLoop && SystemUtils.IS_OS_LINUX;
```

Comment:
minor nit....can you please rename `isLinuxOS` to something more fitting, like `epollEnabled`?